proto/ibc/applications/interchain_accounts/genesis/v1/genesis.proto

```diff
@@ -1,8 +1,8 @@
 syntax = "proto3";
 
-package ibc.applications.interchain_accounts.v1;
+package ibc.applications.interchain_accounts.genesis.v1;
```

Comment:
We don't want to change the package name ideally since it can have unintended side affects (it is used in decoding Anys)

Reply:
wouldnt changing this back to just the ica v1 package cause proto compile errors?

Reply:
No, the go option specifies where the go files are generated

Reply:
i think we can't only change the go option without changing the package name, that causes the matching error i pasted below:

Reply:
yea that's correct, looks like proto doesn't allow multiple go package options in the same package. Did not know that